check_sites.py

```diff
@@ -145,12 +146,14 @@ def main():
         except Exception as inst:
             error_texts.append(inst)
     # This message allows us to check the succes of our action
-    if rws_checker.error_list or error_texts:
+    if rws_checker.error_list or error_texts or rws_checker.warning_texts:
         for checker_error in rws_checker.error_list:
             print(checker_error)
         for error_text in error_texts:
             print(error_text)
-    else:
+        for warning in rws_checker.warning_texts:
+            print(warning)
```

Comment:
This is getting complicated/repetitive; consider extracting the things we want to print:

```python
outputs = rws_checker.error_list + error_texts + rws_checker.warning_texts
if outputs:
  for output in outputs:
    print(output)
else:
  print("success", end="")
```

Note that this no longer repeats each list twice.